lib/sms/sms.c

```diff
@@ -135,8 +136,18 @@ static int sms_cmt_notif_save(void)
 	return 0;
 }
 
+static void sms_ack_work(struct k_work *work)
+{
+	/* Acknowledge the SMS PDU in any case. */
```

Comment:
This comment can now be removed IMO, given the name and short nature of the function